docs/maintainers/determining-concepts.md

```diff
@@ -0,0 +1,11 @@
+# Maintainers - Choosing and defining Concepts
+
+It is worth reading the start of the [Concept Exercises](concept-exercises.md) file to familiarise yourself with Concept Exercises and why writing out concepts is important.
+
+To know which Concept Exercises should exist, a list of the language's concepts should first be compiled. There are various resources to use when compiling this list: books, (official) documentation, your own experiences and the [concepts listed in this repository](../reference/concepts/README.md).
+
+As many languages use the same concepts (but possibly implemented differently), it can greatly help to look at what other languages have already done in this regard. See the [C# concepts](../languages/csharp/reference/concepts/README.md) for an example.
```

Comment:
```suggestion
As many languages use the same concepts (but possibly implemented differently), it can greatly help to look at what other languages have already done in this regard. See the [C# concepts](../languages/csharp/reference/README.md) for an example.
```